msf_updates/src/position_pose_msf/position_pose_sensormanager.h

```diff
@@ -186,8 +186,15 @@ class PositionPoseSensorManager : public msf_core::MSF_SensorManagerROS<
     Eigen::Quaterniond yawq(cos(yawinit / 2), 0, 0, sin(yawinit / 2));
     yawq.normalize();
 
-    q = yawq;
-    q_wv = (q * q_ic * q_vc.conjugate()).conjugate();
+    q_wv = yawq.conjugate();
+
+    if (q_vc.w() == 1) {  // If there is no pose measurement, only apply q_wv.
+      q = q_wv;
+    } else {  // If there is a pose measurement, apply q_ic and q_wv to get initial attitud
+      q = (q_ic * q_vc.conjugate() * q_wv).conjugate();
```

Comment:
This is not correct:
```
(q_ic * q_vc.conjugate() * q_wv).conjugate();
```
since the last conjugate needs to be applied to q_wv and not to the entire expr.
```
q_ic * q_vc.conjugate() * q_wv.conjugate();
```